Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     "packages/kernel-6.1",
     "packages/kmod-5.15-nvidia",
     "packages/kmod-6.1-nvidia",
+    "packages/kmod-6.1-r570-nvidia",
```

Comment:
I was expecting this to be `kmod-6.1-nvidia-r570` to make sorting nice where all the packages that are related should end up listed together where in the future it might look like this:
```
kmod-6.1-nvidia-r570
kmod-6.1-nvidia-r585
kmod-6.12-nvidia-r570
kmod-6.12-nvidia-r585
kmod-6.12-xtrakool-r42 # (made up but shows my point)
```